Создана главная страница и меню

k3ssL opened the pull request.

BlackSoulHub reviewed (changes requested):
Так же @Lx1Cx добавь в этот PR базовый роутинг

k3ssL committed:
Адаптация

k3ssL committed:
Адаптация

BlackSoulHub reviewed (changes requested):
Детальное ревью проведёт @Lx1Cx. Не забудь просмотреть итоговую вёрстку

k3ssL committed:
Ренейм, добавление метода withAuth()

k3ssL committed:
header, menu добавлены в rootLayout

k3ssL committed:
Div заменен на Link

k3ssL committed:
Компонент переписан для использования на разных страницах

k3ssL committed:
Fix компонента

k3ssL committed:
Добавление относительной единицы

k3ssL committed:
Изменен интерфейс

k3ssL committed:
Компонент может принимать данные из массива

k3ssL committed:
Сделан перебор массива и передача данных в компонент книги, изменены стили для самого компонента

k3ssL committed:
Div изменен на Link(Решение конфликтов)

k3ssL committed:
Изменение стилей

k3ssL committed:
Menu перенесено в RootLayout

k3ssL committed:
Изменена версия API

k3ssL committed:
Заменен placeholder

k3ssL committed:
Изменен интерфейс

k3ssL committed:
Изменен отступ, добавлен position

k3ssL committed:
Создан компонент layout

k3ssL committed:
Убраны лишние стили

k3ssL committed:
Fix

k3ssL committed:
Добавлен интерфейс, изменены пропсы

Lx1Cx committed:
Пока пускай будет так

Lx1Cx committed:
Рефакторинг

Lx1Cx committed:
Объединено в группу где будет меню

Lx1Cx committed:
Рефакторинг

k3ssL committed:
Ренейм

k3ssL committed:
Merge remote-tracking branch 'origin/feature/MainPage' into feature/MainPage

Lx1Cx committed:
seo

BlackSoulHub committed:
fix conflicts

# Conflicts:
#	src/app/consts.ts
#	src/app/globals.css
#	src/app/layout.tsx
#	src/entities/Book/api/BookApi.ts
#	src/entities/Book/service/BookService.ts
#	src/pages/MainPage/MainPage.tsx

BlackSoulHub committed:
fix fix

BlackSoulHub committed:
refactoring

Lx1Cx committed:
Рефакторинг

Lx1Cx committed:
Merge remote-tracking branch 'origin/feature/MainPage' into feature/MainPage

# Conflicts:
#	src/pages/admin/tags/CreateTagPage/UI/CreateForm.tsx

Lx1Cx committed:
Рефакторинг

BlackSoulHub committed:
style fix, fix warnings

BlackSoulHub committed:
fix warning
